pkg/task/task.go

```diff
@@ -45,8 +45,10 @@ type Result struct {
 // state of a running or scheduled task.
 type Task struct {
 	Version  int          `json:"version"`  // Schema version
-	Priority int          `json:"priority"` // scheduling priority
+	Priority int          `json:"priority"` // Scheduling priority
 	ID       string       `json:"id"`       // Unique identifier for this task
+	Plan     string       `json:"plan"`     // Test plan
+	Case     string       `json:"case"`     // Test case
```

Comment:
Maybe these two would move to a `Metadata` struct, so that we include more relevant metadata (such as `CreatedBy`, `Commit`, `Branch`, `Repo`, `Dependencies`, etc.

I imagine some of these won't be filled by all users, but if they are, it would be nice to be able to filter based on them...

Reply:
I think that'd be a great idea and give us more flexibility in the future.